crates/krun-sys/build.rs

```diff
@@ -1,18 +1,29 @@
 use std::env;
 use std::path::PathBuf;
 
-fn main() {
+fn main() -> Result<(), pkg_config::Error> {
     println!("cargo::rerun-if-changed=wrapper.h");
-    println!("cargo::rustc-link-lib=krun");
+
+    let library = pkg_config::probe_library("libkrun")?;
+
+    println!("cargo:warning={:?}", library.include_paths);
```

Comment:
```suggestion
    println!("cargo::warning=include paths: {include_paths:?}", include_paths = library.include_paths);
```

(If this is meant to stay...)

Reply:
Oops, that shouldn't be there. I've just removed it, thanks!